website/content/docs/configuration/worker/pki-worker.mdx

```diff
@@ -78,6 +78,29 @@ kms "aead" {
 workers. These fields are only valid for [KMS Workers][]. `name` and
 `description` can only be set for PKI workers through the API.
 
+# Multi-Hop Workers
```

Comment:
```suggestion
# Multi-Hop Workers<sup>HCP Only</sup>
```

Reply:
Adding the "HCP Only" tag as the multi hop session functionality is only for HCP users